lib/honeybadger/notice_filter/default.ex

```diff
@@ -19,6 +19,7 @@ defmodule Honeybadger.NoticeFilter.Default do
     |> apply_filter(:context, &filter.filter_context/1)
     |> apply_filter(:params, &filter.filter_params/1)
     |> apply_filter(:cgi_data, &filter_cgi_data/1)
+    |> apply_filter(:cgi_data, &filter.filter_cgi_data/1)
```

Comment:
This would go in the filter_cgi_data function below. Oddly, the filter module isn't passed in there so it looks it up again.

Reply:
Ah, I see. I'm not sure what the correct fix is, then.

Reply:
@joshuap I pushed an update to this branch that should fix it.